impeller/entity/contents/sweep_gradient_contents.cc

```diff
@@ -0,0 +1,107 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sweep_gradient_contents.h"
+
+#include "flutter/fml/logging.h"
+#include "impeller/entity/contents/content_context.h"
+#include "impeller/entity/entity.h"
+#include "impeller/renderer/render_pass.h"
+#include "impeller/tessellator/tessellator.h"
+
+namespace impeller {
+
+SweepGradientContents::SweepGradientContents() = default;
+
+SweepGradientContents::~SweepGradientContents() = default;
+
+void SweepGradientContents::SetPath(Path path) {
+  path_ = std::move(path);
+}
+
+void SweepGradientContents::SetCenterAndAngles(Point center,
+                                               Scalar start_angle,
+                                               Scalar end_angle) {
```

Comment:
Here and everywhere else angles are handled, prefer to use the special unit wrappers `Radians` and/or `Degrees` rather than `Scalar` (both for passing over API boundaries and storage).